src/OpenLoco/src/Vehicles/Vehicle.cpp

```diff
@@ -686,6 +686,78 @@ namespace OpenLoco::Vehicles
         return AirportObjectFlags::acceptsHeavyPlanes;
     }
 
+    // 0x004AFFF3
+    VehicleBogie* VehicleBogie::flipCar()
+    {
+        if (getSubType() != VehicleEntityType::bogie)
```

Comment:
If the function can be called on non bogie then it stands to reason it shouldn't be a bogie function. What you could do is split it in two here and have a function that takes a base and if it is a bogie then call this bogie func

Reply:
This was added while I was trying to debug the CarComponent constructor, so I will remove it